firstvoices/backend/management/commands/convert_draftjs_to_html.py

```diff
@@ -0,0 +1,102 @@
+import json
+import logging
+
+from django.core.management.base import BaseCommand
+from draftjs_exporter.html import HTML
+
+from backend.models import Site, Song, Story, StoryPage
+from backend.models.widget import SiteWidget, WidgetSettings
+
+
+class Command(BaseCommand):
+    help = "Convert fv-be draftjs content to html content."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--sites",
+            dest="site_slugs",
+            help="Site slugs of the sites to convert draftjs content for, separated by comma (optional)",
+            default=None,
+        )
+
+    @staticmethod
+    def is_draftjs_content(text):
+        try:
+            content = json.loads(text)
+            return (
+                isinstance(content, dict)
+                and "blocks" in content
+                and "entityMap" in content
+            )
+        except json.JSONDecodeError:
+            return False
+
+    def convert_draftjs_to_html(self, text):
+        if self.is_draftjs_content(text):
+            exporter = HTML()
+            return exporter.render(json.loads(text))
+        return text
+
+    def handle(self, *args, **options):
+        logger = logging.getLogger("convert_draftjs_to_html")
```

Comment:
To make our logs more predictable, please use `logging.getLogger(__name__)` and just leave it at the default level